yarn.lock

```diff
@@ -8,18 +8,18 @@
   integrity sha512-RiMAVpSluRbWb2hlT9wMJ0r2l+MUZzScYjY+w2iWRzjOr9Zzzs4tYzJT6Sd94PDz3LzCuf4aGOwS6pkKXTEBLw==
 
 "@analytics/storage-utils@^0.2.2":
-  version "0.2.3"
```

Comment:
Can't we revert this file?

Reply:
Yep I was rebuilding it as well - just merged with `next` so new angular is comming here as well